lib/api/v3/work_packages/schema/work_package_schema.rb

```diff
@@ -63,19 +59,19 @@ def assignable_statuses_for(user)
           end
 
           def assignable_versions
-            @work_package.assignable_versions if defines_assignable_values?
+            @work_package.assignable_versions unless @work_package.nil?
           end
 
           def assignable_priorities
-            @work_package.assignable_priorities if defines_assignable_values?
+            IssuePriority.where(active: true)
```

Comment:
Note to self and whoever reads this. It would be better to have an `active` scope on IssuePriority to avoid AR methods here. I am aware, that the code was just moved, though.